website_product_name/views/templates.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <!-- website name shown in list/grid -->
+        <template id="products_website_name"
+            inherit_id="website_sale.products_item"
+            name="Product website name">
+            <xpath expr="//a[@itemprop='name']" position="attributes">
+                <attribute name="t-field">product.display_website_name</attribute>
+            </xpath>
+        </template>
+        <!-- website name in product page -->
+        <template id="website_name_product_page" inherit_id="website_sale.product"  name="Product website name for product page">
+            <xpath expr="//t[@t-set='additional_title']" position="attributes">
+                <attribute name="t-value">product.display_website_name</attribute>
+            </xpath>
+            <xpath expr="//ol[@class='breadcrumb']/li[@class='active']" position="replace">
+                <li class="active"><span t-field="product.display_website_name"/></li>
+            </xpath>
+            <xpath expr="//div[@id='product_details']//h1[@t-field='product.name']" position="attributes">
+                <attribute name="t-field">product.display_website_name</attribute>
+            </xpath>
+        </template>
+
+
+        <template id="website_name_product_recommended_ext" inherit_id="website_sale.recommended_products"  
+            name="Product website name for recomendations">
+            <xpath expr="//h5/a/span" position="attributes">
+                <attribute name="t-att-title">alt_product.display_website_name</attribute>
+                <attribute name="t-field">alt_product.display_website_name</attribute>
+            </xpath>
+        </template>
+        <!-- Show website name in cart too -->
+        <template id="products_website_name_cart" inherit_id="website_sale.cart_lines"  name="Product website name in cart">
+            <xpath expr="//td[@t-if='line.product_id.product_tmpl_id']//div" position="replace">
```

Comment:
wouldn't be better to select the `<a />` element directly?

Reply:
in my original project that element  had been changed a bit. Replacing the entire <a> even if it meant repeating a line of code was more convenient.